moped-editor/src/components/GridTable/FiltersChips.js

```diff
@@ -33,19 +45,56 @@ const FiltersChips = ({ filters, isOr, filtersConfig }) => {
       (fieldConfig) => fieldConfig.name === filter.field
     );
     const fieldOperatorConfig = filtersConfig.operators[filter.operator];
+    const isDateType = fieldFilterConfig.type === "date";
     return {
       filterLabel: fieldFilterConfig?.label,
       operatorLabel: fieldOperatorConfig?.label,
-      filterValue: filter.value,
+      filterValue: isDateType ? formatDateType(filter.value) : filter.value,
     };
   });
 
+  /**
+   * Triggered by Filter Chip delete click
+   * Removes specified filter from state and update the url params
+   * @param {string} filterIndex - The index of the filter to be deleted
+   */
+  const handleDeleteButtonClick = (filterIndex) => {
```

Comment:
part of me thinks that the contents of this function could be defined in the parent component and then passed down. Might be good for reuse but I'm not sure that will ever happen. It would mean less props needed in this component. I don't really feel strongly either way.

Reply:
nah - let's ship it!